backend/open_api_spec.yml

```diff
@@ -0,0 +1,729 @@
+                      access_token:
+                        type: string
+                        required: true
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+
+  /curation/collections/{collection_id}/datasets/status:
+    get:
+      description: Get the processing status for a Dataset.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/optional_query_curator_tag"
+        - $ref: "#components/parameters/optional_query_dataset_uuid"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type:
+                  object:
+                    properties:
+                      status:
+                        type: string
+                        enum: [FAILURE, PENDING, SUCCESS]
+
+components:
+  schemas:
+    user_uuid:
+      description: A unique identifier of a logged in User of Corpora.
+      type: string
+    collection_uuid:
+      description: A unique identifier of a Collection.
+      type: string
+    curator_tag:
+      description: A curator-provided tag that is unique within a Collection.
+      type: string
+    dataset_uuid:
+      description: A unique identifier of a Dataset.
+      type: string
+    geneset_uuid:
+      description: A unique identifier of a gene set.
+      type: string
+    visibility:
+      description: >-
+        Determines the visibility of the collection, that will either make the collection visible on the public
+        sites or only viewable through obfuscated URLs.
+      type: string
+      enum: [PUBLIC, PRIVATE]
+    distribution:
+      description: >-
+        Cellxgene runs a heuristic to detect the approximate distribution of the data in X so that it can accurately
+        calculate statistical properties of the data. This field enables the curator to override this heuristic
+        and specify the data distribution explicitly.
+      type: string
+      enum: [COUNT, NORMAL]
+    is_primary_data:
+      description: >-
+        Describes whether cellular observations for this dataset are all canonical (PRIMARY),
+        all non-canonical (SECONDARY), or contain a mixture (BOTH).
+      type: string
+      enum: [PRIMARY, SECONDARY, BOTH]
+    links:
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        required:
+          - link_url
+          - link_type
+        properties:
+          link_name:
+            type: string
+          link_url:
+            type: string
+          link_type:
+            type: string
+            enum: [PROTOCOL, RAW_DATA, DOI, LAB_WEBSITE, OTHER, DATA_SOURCE]
+    dataset:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - name
+        - revision
+        - collection_id
```

Comment:
Is there a need to duplicate the `collection_id` in the dataset? (This currently occurs in the portal API)